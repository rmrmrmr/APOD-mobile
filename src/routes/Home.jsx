/* eslint-disable */
import Searchbar from "../components/Searchbar";
import Logo from "../components/Logo";

const Home = () => {
  return (
    <div className="h-screen">
      <Searchbar />
      <Logo />
    </div>
  );
};

export default Home;
