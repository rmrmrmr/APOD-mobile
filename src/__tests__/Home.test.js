import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from '../components/Logo';

it('renders correctly', () => {
  const home = renderer
    .create(<Router><Home /></Router>)
    .toJSON();
  expect(home).toMatchSnapshot();
});
