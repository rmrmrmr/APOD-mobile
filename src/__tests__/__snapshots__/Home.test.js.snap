// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-blue-400 flex items-center justify-center h-2/6"
>
  <img
    alt="img not found"
    className="px-20"
    src="../img/logo.png"
  />
</div>
`;
